haystack/document_stores/memory.py

```diff
@@ -64,6 +65,8 @@ def __init__(
                                    you have at least `embedding_dim`*`scoring_batch_size`*4 bytes available in GPU memory.
                                    Since the data is originally stored in CPU memory there is little risk of overruning memory
                                    when running on CPU.
+        :param devices: List of GPU devices to limit inference to certain GPUs and not use all available ones (e.g. [torch.device('cuda:0')]).
```

Comment:
Does it mean the following:
List of GPU devices you want to limit inference to so as not to use all available GPUs. Example: `[torch.device(`cuda:0`)]`.